Header header
string file_name
float32[6] transformation # x,y,z,yaw,pitch,roll. meters and degrees.
bool overlay # dont clear the previous pointcloud 